docs/azure_arc_jumpstart/azure_arc_data/kubeadm/kubeadm_azure_postgresql_arm_template/_index.md

```diff
@@ -0,0 +1,230 @@
+---
+type: docs
+title: "PostgreSQL ARM Template"
+linkTitle: "PostgreSQL ARM Template"
+weight: 3
+description: >
+---
+
+## Deploy Azure Arc-enabled PostgreSQL in directly connected mode on Cluster API Kubernetes cluster with Azure provider using an ARM Template
+
+The following Jumpstart scenario will guide you on how to deploy a "Ready to Go" environment so you can start using [Azure Arc-enabled data services](https://docs.microsoft.com/azure/azure-arc/data/overview) deployed on [Kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/) Kubernetes cluster.
+
+By the end of this scenario, you will have a Kubeadm Kubernetes cluster deployed with an Azure Arc Data Controller and a Microsoft Windows Server 2022 (Datacenter) Azure client VM, installed & pre-configured with all the required tools needed to work with Azure Arc-enabled data services.
+
+> **NOTE: Currently, Azure Arc-enabled data services with PostgreSQL is in [public preview](https://docs.microsoft.com/azure/azure-arc/data/release-notes)**.
+
+## Prerequisites
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- [Install or update Azure CLI to version 2.36.0 and above](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+- [Generate SSH Key](https://docs.microsoft.com/azure/virtual-machines/linux/create-ssh-keys-detailed) (or use existing ssh key).
+
+- Create Azure service principal (SP). To deploy this scenario, an Azure service principal assigned with the following RBAC role is required:
+
+  - "Owner" - Required for provisioning Azure resources, installing Cloud Defender Azure-Arc enabled Kubernetes extension and dismiss alerts, being able to view Azure-Arc enabled Kubernetes Cloud Defender extension findings, being Azure Arc-enabled data services billing, monitoring metrics, and logs management and creating role assigment (Monitoring Metrics Publisher).
```

Comment:
```suggestion
  - "Owner" - Required for provisioning Azure resources, installing the Azure-Arc enabled Kubernetes Microsoft Cloud Defender for Containers cluster extension (incl. dismissing alerts and being able to view  findings), interact with Azure Arc-enabled data services billing, monitoring metrics, and logs management and creating role assignment for the Monitoring Metrics Publisher role.
```

Reply:
So I made suggestion but we should consider removing the Microsoft Cloud Defender verbose since you are not installing it in the scenario and might cause confusion.